exercises/forth/forth_test.go

```diff
@@ -0,0 +1,53 @@
+package forth
+
+// API:
+// func Forth([]string) ([]int, error)
+//
+
+import "testing"
+
+const targetTestVersion = 1
+
+type testCase struct {
+	description string
+	input       []string
+	expected    []int // nil slice indicates error expected.
+}
+
+func sliceEqual(a, b []int) bool {
+	if len(a) != len(b) {
+		return false
+	}
+	for i := range a {
+		if a[i] != b[i] {
+			return false
+		}
+	}
+	return true
+}
+
+func TestForth(t *testing.T) {
+	if testVersion != targetTestVersion {
+		t.Fatalf("Found testVersion = %v, want %v", testVersion, targetTestVersion)
+	}
+	for _, tc := range testCases {
+		v, err := Forth(tc.input)
```

Comment:
a bit of history here. Back in https://github.com/exercism/xgo/issues/275#issuecomment-206948940 , we decided to make sure that whenever a function asks for an error that we make sure in tests that it is an error by saying `var err error`. Now that effort is completed for current exercises.

Now it is time to decide, do we still want to do that for this exercise? Or do we assume that by the time the student gets to this point the student will probably not do something weird like return something of a different type? It's admitted by me that it's a weird thing to do since it would be unlikely to be in the test suite of a non-TDD project.